src/test/java/com/paritytrading/foundation/ByteArraysTest.java

```diff
@@ -111,4 +111,9 @@ public void unpackLongWithAllocation() {
         assertArrayEquals(new byte[] { 0x01, 0x02, 0x03, 0x00, 0x00, 0x00, 0x00, 0x00 }, a);
     }
 
+    @Test
+    public void roundtrip() {
```

Comment:
Do you need to test this with for `long` as well?

Reply:
I added explicit tests for all types: short, integer, and long.